eclass/toolchain-funcs.eclass

```diff
@@ -13,9 +13,8 @@
 # in such a way that you can rely on the function always returning
 # something sane.
 
-case ${EAPI:-0} in
-	# EAPI=0 is still used by crossdev, bug #797367
-	0|5|6|7|8) ;;
+case ${EAPI} in
+	6|7|8) ;;
```

Comment:
Can you double check crossdev is fine with this? (Just do a test run with some arbitrary CHOST)

Reply:
With crossdev-20220205 ...
```
--------------------------------------------------------------------------------
 * crossdev version:      20220205
 * Host Portage ARCH:     amd64
 * Host Portage System:   x86_64-pc-linux-gnu (i686-pc-linux-gnu x86_64-pc-linux-gnu)
 * Target Portage ARCH:   arm64
 * Target System:         aarch64-unknown-linux-gnu
 * Stage:                 4 (C/C++ compiler)
 * USE=multilib:          no
 * Target ABIs:           arm64
[...]
```

... and with crossdev-20210608 (to check whether the old version actually fails):
```
 * error: : EAPI 0 not supported
```
(Why doesn't it show the name of the eclass in the error message? Looks like there are more problems than just EAPI.)
```
 * 
 * If you file a bug, please attach the following logfiles:
 * /var/log/portage/cross-x86_64-pc-linux-gnu-info.log
 * error: : EAPI 0 not supported
 * 
 * If you file a bug, please attach the following logfiles:
 * /var/log/portage/cross-aarch64-unknown-linux-gnu-info.log
[...]
```

BTW, I had some trouble finding an old distfile for crossdev. Apparently, somebody thought it wise to remove them all. For example, https://dev.gentoo.org/~slyfox/distfiles/crossdev-20210608.tar.xz is gone. It isn't on mirrors either.

@gentoo/crossdev

Reply:
(Re distfiles: they get removed/archived when a dev retires, for good or ill. I worry about that in terms of longevity too. I've archived all of slyfox's at https://dev.gentoo.org/~sam/distfiles/ (open directory).)

Reply:
Are they actually archived? I thought no backups were done of home directories on woodpecker?